### PR TITLE
Kaw 7904 header

### PR DESCRIPTION
**Pull request by TomaszDziezykNetcentric**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #KAW-7904

Test URLs:
- Before: https://main--edge-delivery-bimota--netcentric.hlx.page/drafts/tdziezyk/test
- After: https://KAW-7904-header--edge-delivery-bimota--netcentric.hlx.page/drafts/tdziezyk/test

### CODE REVIEW

**Comment by IoanaOrca** [comment]
Screenshot at Sep 12 15-03-36
Hover color in the designs is darker red, are we keeping the lighter one?
Screenshot 2024-09-12 at 15 10 16
I see in the design it's mentioned that the flyout is closing when the user is clicking on the darker part. Are we also adding that feature?
In the screenshot you can see  the flyout staying open when it's not closed, i assume we add that later.
Screenshot 2024-09-12 at 15 17 58
And one last thing. I see int he designs that there is an image animation on hover. Do we have an example of content for that?

File: scripts/scripts.js [hunk 1]
```diff
@@ -94,6 +94,36 @@ function swappingPlacesBlock(main) {
   });
 }
 
+export function customDecoreateIcons(main) {
```
Comment (IoanaOrca) [inline scripts/scripts.js:97]: awesome 👏

File: blocks/header/header.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import { getMetadata } from '../../scripts/aem.js';
+import { customDecoreateIcons } from '../../scripts/scripts.js';
 import { loadFragment } from '../fragment/fragment.js';
 
 // media query match that indicates mobile/tablet width
-const isDesktop = window.matchMedia('(min-width: 992px)');
+const isDesktop = window.matchMedia('(min-width: 1024px)');
```
Comment (IoanaOrca) [inline blocks/header/header.js]: i think this is 1025 as per https://ncchat.slack.com/archives/C07KG702WMC/p1726052767589049

**Comment by lakshmishriaswathnarayana** [comment]
Yes like @IoanaOrca mentioned, when the nav bar below is opened, the bikes should have some animation. Can you align with michael tomorrow ?

**Review by IoanaOrca — APPROVED** [review]
Looks great!
Awesome job.

**Pull request merged**